### ISSUE TITLE
Heat Stroke and Frostbite damage too low

### ISSUE THREAD

**Issue by AeiouEnigma** [anIceyGirl/SurvivalOverhaul#6]
The damage from Heat Stroke and Frostbite is low enough and infrequent enough that a player can simply eat food to mitigate the damage via natural regeneration. You might consider making these effects more deadly so that players see them as serious threats to be avoided.

**Comment by anIceyGirl** [comment]
I'd be willing to make the damage configurable, if that's what you're after. It's perfectly fine by my standards, since, for all intents and purposes, having to constantly eat to keep your health up is not going to work forever. But I understand if you want it to be configurable.

**Comment by AeiouEnigma** [comment]
Yeah it's definitely a subjective take, but configurable damage (or configurable time between damage ticks) would be a welcome addition :)
